openhands_resolver/send_pull_request.py

```diff
@@ -367,14 +368,22 @@ def process_single_issue(
     make_commit(patched_repo_dir, resolver_output.issue, issue_type)
 
     if issue_type == "pr":
+        # Parse the explanations and create a summary message
+        explanations = json.loads(resolver_output.success_explanation)
+        summary_message = "I have made updates to address the review comments:\n\n"
+        for explanation in explanations:
+            summary_message += f"- {explanation}\n"
+        
```

Comment:
This should be summarized using an LM, not listed verbatim. To have an example of how to make a call to a language model within this repository, you can look here:

https://github.com/All-Hands-AI/openhands-resolver/blob/25d74a2da4b011e8f024e20307229811302e557e/openhands_resolver/issue_definitions.py#L134-L139

Reply:
Openhands fix success summary


The AI has successfully addressed the feedback by implementing LLM-based summarization for PR review comments. The changes include adding LLM configuration support, modifying the comment processing logic to use language models for summarization, and ensuring proper integration throughout the codebase. The key points indicate they've moved away from verbatim listing of comments to using LLM-based summarization, which directly addresses the original feedback request. The implementation references the example provided in the feedback link by making the necessary structural changes to support LLM calls within the repository.